docs/user-guide/scheduler.md

```diff
@@ -1034,6 +1034,10 @@ node you must be aware of a few things:
  - The `srun` command must specify any Slurm options that differ in value
    from those specified to `sbatch`. This typically means that you need 
    to specify the `--nodes`, `--ntasks` and `--tasks-per-node` options to `srun`.
+ - On the ARCHER2 full system, you will need to include the `--oversubscribe` 
+   flag to your `srun` command, and you will need to define the memory required 
+   by each subjob with the `--mem=<amount of memory>` flag. The amount of memory 
+   is given in MiB.
```

Comment:
I think this should be "The amount of memory is given in MiB by default but other units can be specified. If you do not know how much memory to specify, we recommend that you specify 1500M (1,500 MiB)."